contrib/basicauth/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+nginx:
+  build: "nginx"
+  ports:
+    - "5441:5441"
+  links:
+    - registryv2:registryv2
+registryv2:
+  build: "../../"
+  ports:
+    - "5000"
```

Comment:
That's the port for the nginx container. This is the port for the registry. If the nginx container is intended to proxy *all* traffic for the registry, then the registry itself should not be published to the host (as that would allow direct access to the registry, skipping the nginx server and authentication).

I may be wrong here, so better check if I'm not talking nonsense here :smile: